app/services/rapid_ready_for_decision/views/shared/medications.erb

```diff
@@ -10,13 +10,13 @@
 </p>
 
 <% if @assessed_data[:medications].present? %>
+  <p> <br /> </p>
```

Comment:
Unrelated to the main point of the PR but this fixes a small spacing issue that's been bugging me

Reply:
I think somewhere here we want to include an extra section for the asthma claims:

> Important! Medications that may have been prescribed for asthma are marked with ►arrows◀︎. Claims adjudicators should check the patient’s medical records to see what the medication was prescribed for. 
See “How we identify asthma prescriptions” at the end of this document for more details.